src/App.js

```diff
@@ -6,16 +6,43 @@ import { History } from "./pages/history/history";
 import { Home } from "./pages/home/home";
 import { Playlist } from "./pages/playlist/playlist";
 import { WatchLater } from "./pages/watchLater/watchLater";
+import { Routes, Route } from 'react-router-dom';
+import { LikedPage } from "./pages/likedVideo/likedVideo";
+import { RequireAuth } from "./components/requireAuth/requireAuth";
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
```

Comment:
Create index file and Exports files from there.

Reply:
ok, i will do that, thank you